staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode_test.go

```diff
@@ -37,6 +37,11 @@ func TestDecode(t *testing.T) {
 		return b
 	}
 
+	type s struct {
+		LowerA int `cbor:"a"`
+		UpperB int `cbor:"B"`
+	}
```

Comment:
Nit: This is only used for 3 cases, and two of them only use the first field. To make it possible to understand those cases without jumping to the definition of `s`, those cases may as well include the struct types as part of the value for `into`, and exclude UpperB in the two tests that only use LowerA.